.metadata/.plugins/org.eclipse.cdt.make.core/specs.c

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think you opened the repo in eclipse or something. Please do not commit all these other extraneous files for your other projects. This will clutter the repo.